build-script.py

```diff
@@ -343,27 +344,36 @@ def generate_gyb_files(
 
 
 def run_code_generation(
-    toolchain: str, verbose: bool, swiftsyntaxbuilder_destination: str
+    swiftbasicformat_destination: str,
+    swiftsyntaxbuilder_destination: str,
+    toolchain: str,
+    verbose: bool
 ) -> None:
     print("** Running code generation **")
 
-    swift_exec = os.path.join(toolchain, "bin", "swift")
+    target_to_dest_dir = {
+        "generate-swiftbasicformat": swiftbasicformat_destination,
+        "generate-swiftsyntaxbuilder": swiftsyntaxbuilder_destination,
+    }
 
-    swiftpm_call = [
-        swift_exec, 'run',
-        "--package-path", os.path.join(PACKAGE_DIR, 'Code-Generation'),
-        "generate-swift-syntax-builder", swiftsyntaxbuilder_destination
-    ]
-    
-    if verbose:
-        swiftpm_call.extend(["--verbose"])
+    for target, destination_dir in target_to_dest_dir.items():
+        swift_exec = os.path.join(toolchain, "bin", "swift")
 
-    env = dict(os.environ)
-    env["SWIFT_BUILD_SCRIPT_ENVIRONMENT"] = "1"
-    # Tell other projects in the unified build to use local dependencies
-    env["SWIFT_SYNTAX_PARSER_LIB_SEARCH_PATH"] = \
-        os.path.join(toolchain, "lib", "swift", "macosx")
-    check_call(swiftpm_call, env=env, verbose=verbose)
+        swiftpm_call = [
+            swift_exec, 'run',
+            "--package-path", os.path.join(PACKAGE_DIR, 'Code-Generation'),
+            target, destination_dir
+        ]
+        
+        if verbose:
+            swiftpm_call.extend(["--verbose"])
+
+        env = dict(os.environ)
+        env["SWIFT_BUILD_SCRIPT_ENVIRONMENT"] = "1"
+        # Tell other projects in the unified build to use local dependencies
+        env["SWIFT_SYNTAX_PARSER_LIB_SEARCH_PATH"] = \
```

Comment:
Can we use `SwiftParser` now for the code generation?

Reply:
What do you mean by using `SwiftParser` for code generation? We are using `SwiftSyntaxBuilder` and I don’t see a reason why `SwiftSyntaxBuilder` shouldn’t be able to use `SwiftParser` as well.

Reply:
I should have been more specific: can we remove the `env["SWIFT_SYNTAX_PARSER_LIB_SEARCH_PATH"]` bit?

Reply:
Oh, good catch. We can absolutely do that because `CodeGeneration` doesn’t depend on neither the old nor the new parser.